docs/handbook/tutorial.rst

```diff
@@ -358,6 +415,9 @@ Applying point transforms
     # multiply each pixel by 1.2
     out = im.point(lambda i: i * 1.2)
 
+.. image:: transformed_hopper.jpg
+    :align: center
```

Comment:
```python
from PIL import Image
im = Image.open("Tests/images/hopper.png")
out = im.point(lambda i: i * 1.2)
out.save("transformed_hopper.jpg")
```
gives me

![transformed_hopper](https://github.com/user-attachments/assets/0636a152-a3dc-4e65-91bc-8419a23c72fb)

Yours looks different?
![transformed_hopper_pr](https://raw.githubusercontent.com/python-pillow/Pillow/c337dfe98a0097f8f1999bb6c2bf12804a2d0552/docs/handbook/transformed_hopper.jpg)

Reply:
@aclark4life Same for me, does this need fixing?

Reply:
Sorry, fixed in 96fa1f5dbf43fe26a9b7abd98697ae46e8165435!